Guide.docc/CommonProblems.md

```diff
@@ -393,3 +393,312 @@ struct LivingIsland: Feedable {
 Here, a new type has been created that can satisfy the needed inheritance.
 Incorporating will be easiest if the conformance is only used internally by
 `Island`.
+
+## Crossing Isolation Boundaries
+
+Any value that needs to move from one isolation domain to another
+must be `Sendable`.
```

Comment:
Should we say "must either be `Sendable` or must preserve mutually exclusive access", or similar? It's no longer the case that values that move between isolation domains must _always_ have a type that conforms to `Sendable`.

Reply:
Yeah, exellent point.